docs/mozillavpn.json

```diff
@@ -103,6 +103,11 @@
       "build": null,
       "date": "2022-11-14",
       "platforms": ["ANDROID", "IOS", "LINUX", "MACOS", "WINDOWS"]
+    },
+    "2.11.1": {
+      "build": null,
+      "date": "2022-11-24",
+      "platforms": ["MACOS", "WINDOWS"]
```

Comment:
or alternatively, let's remove MACOS from here.

Reply:
I figured that since the artifacts got published, we might as well include them in the manifest, even if the latest build we intend on releasing for MacOS is 2.11.0

Reply:
ok!